kitsune/products/views.py

```diff
@@ -35,6 +37,16 @@ def product_landing(request, template, slug):
         return HttpResponse(json.dumps({'topics': topic_list}),
                             content_type='application/json')
 
+    if request.LANGUAGE_CODE == settings.WIKI_DEFAULT_LANGUAGE:
+        show_community_support = True
+    else:
+        try:
```

Comment:
This can be rewritten like this

```python
show_community_support = Document.objects.filter(locale=request.LANGUAGE_CODE, 
                                                 parent__slug='get-community-support').exclude(
                                                 current_revision=None).exists()
```

It will check whether any translated document is exists of the English document having slug ``get-community-support``. It will also exclude any document having no current revision.